mongo/upsert_users.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+# Update users if database has been already created.
+# Users creation on init is left to `init_02_create_user.sh`
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+MONGO_CMD=( mongo --host 127.0.0.1 --port 27017 --quiet )
```

Comment:
I don't think I understand the use of parentheses for `MONGO_CMD`.
```
john@scrappy:~$ MONGO_CMD=( mongo --host 127.0.0.1 --port 27017 --quiet )
john@scrappy:~$ echo $MONGO_CMD
mongo
```

Reply:
Actually, the idea is to use as an array, not a single variable. Look below 
`"${MONGO_CMD[@]}" "$MONGO_INITDB_DATABASE" <<-EOJS` 

I got this from the docker entrypoint of official Mongo docker image. 
https://github.com/docker-library/mongo/blob/master/docker-entrypoint.sh#L287

I found it was a good idea to do it like this.
I think the idea is to avoid bad characters when calling a command with parameters.

Reply:
Oh, cool. Thanks for the explanation.
```
john@scrappy:~$ MONGO_CMD=( mongo --host 127.0.0.1 --port 27017 --quiet )
john@scrappy:~$ echo ${MONGO_CMD[@]}
mongo --host 127.0.0.1 --port 27017 --quiet
```